thorgame/dsr209/thor_codingame.cpp

```diff
@@ -120,18 +105,16 @@ int main()
         // it fails that but it is already in this if statement, then Thor's
         // direction can be determined to be north without the need to do any
         // more comparisons.
-        
-        if(light_x == initial_tx){
-            if(light_y > initial_ty){
+        if (light_x == thor_x) {
```

Comment:
The paragraph above seems unfinished.

Reply:
I added to the end of that paragraph to close it out a little better.